package.json

```diff
@@ -54,7 +54,7 @@
     "@changesets/cli": "2.26.0",
     "@justeat/browserslist-config-fozzie": "1.x",
     "@justeat/eslint-config-fozzie": "5.1.0",
-    "@justeat/pie-design-tokens": "3.7.0",
+    "@justeat/pie-design-tokens": "4.0.0-beta.0",
```

Comment:
Please make sure this doesn't add anything funky to the existing tokens (colour primarily) - I'm sure percy would pick up any regressions though! :D